resources/views/external-link-confirm.blade.php

```diff
@@ -13,13 +13,13 @@
             Livewire.emit('openModal', '{{ $identifier }}')
         }
     }"
-    class="inline-block items-center space-x-2 font-semibold break-all cursor-pointer link"
+    class="items-center font-semibold break-all cursor-pointer link"
     @click="openModal"
 >
     <span>{{ $text ?? $slot ?? '' }}</span>
 
     @unless($noIcon)
-        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 mr-2 ml-1 -mt-1" />
+        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 ml-1 -mt-2" />
```

Comment:
```suggestion
        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 ml-1 -mt-1" />
```

looks better with `-mt-1` I think 🤔 
![image](https://user-images.githubusercontent.com/35610748/126511257-9931b070-461d-4194-8779-5e996e20c474.png)


with `-mt-2`:
![image](https://user-images.githubusercontent.com/35610748/126511215-a708371f-9548-4a07-9a50-efa3d1a5f792.png)

Reply:
Changed for 1.5 
1: 
![image](https://user-images.githubusercontent.com/17262776/126512722-91cbb201-f9e5-42c1-882c-502d4e7c8ff7.png)

1.5:
![image](https://user-images.githubusercontent.com/17262776/126512781-21e9b7a3-04a0-455c-8773-d89c0e1ffc46.png)

Reply:
(mt-1 a little bit unaligned on multiline links)